api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessActivator.java

```diff
@@ -17,9 +17,7 @@
 
 public class LocationBasedAccessActivator extends BaseModuleActivator {
 
-
```

Comment:
Can you just revert all changes in this file?

Reply:
@dkayiwa  Do you mean this lines? Or that person attribute type Creation during the module startup?

Reply:
Such that this file is not among the commit files because you really did not change anything in it.

Reply:
Reverted